pyproject.toml

```diff
@@ -160,12 +160,4 @@ exclude_lines = [
 
 [tool.flake8]
 max-line-length = 100
-extend-ignore = "E203, E501, E741, W503"
-
-[tool.interrogate]
-verbose = 2
-ignore-init-method = true
-ignore-init-module = true
-ignore-magic = true
-ignore-property-decorators = true
-ignore-semiprivate = true
+extend-ignore = "E203, E501, E741, W503"
```

Comment:
Can you remove this newline change?

Reply:
Done. I'll need to find a way to prevent this from happining in vscode...

Reply:
Vim does this correctly :stuck_out_tongue_winking_eye: